src/ASM/LR/ASMu2D.C

```diff
@@ -2675,3 +2675,29 @@ void ASMu2D::getElmConnectivities (IntMat& neigh) const
     }
   }
 }
+
+
+void ASMu2D::getBoundaryElms (int lIndex, int orient, IntVec& elms) const
+{
+  LR::parameterEdge edge;
+  switch(lIndex)
+  {
+  case 1: edge = LR::WEST;  break;
+  case 2: edge = LR::EAST;  break;
+  case 3: edge = LR::SOUTH; break;
+  case 4: edge = LR::NORTH; break;
+  default:edge = LR::NONE;
+  }
+
+  std::vector<LR::Element*> elements;
+  this->getBasis(1)->getEdgeElements(elements, edge);
+  int idx = lIndex < 3 ? 1 : 0;
```

Comment:
`idx` should be inside the lambda-scope if not used outside

Reply:
done.